contracts/airdrops/ExistingTokenDrop.sol

```diff
@@ -3,59 +3,43 @@ pragma solidity ^0.8.14;
 
 import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
-import "@openzeppelin/contracts/access/Ownable.sol";
-import "../interfaces/AirdropInfo.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 import "../interfaces/AirdropMerkleProof.sol";
-import "../interfaces/IAirBroFactory.sol";
 
 /// @title Airdrops existing ERC20 tokens for airdrop recipients
-contract ExistingTokenDrop is AirdropInfo, AirdropMerkleProof, Ownable {
+contract ExistingTokenDrop is AirdropMerkleProof {
```

Comment:
Do you really need to extend `AirdropMerkleProof`? I don't see you using `checkProof` in this contract

Reply:
You are right. I will remove them from all contracts that do not actually implement a MerkleProof check.